### PR TITLE
CHANGE: changed project structure

### PR DESCRIPTION
**Pull request by iryna-novykova-axon**

### CODE REVIEW

File: bladeui/css/blade.scss [hunk 1]
```diff
@@ -1,24 +1,30 @@
...
-@import 'blade/profile.scss';
+
+@import 'settings/about.scss';
+@import 'settings/transfers.scss';
+@import 'settings/profile.scss';
+
+@import 'transfers/manualTransfer.scss';
+@import 'transfers/transfersList.scss';
+
+@import 'pagesForUnregisteredUsers/createPassword.scss';
```
Comment (maksym-dyachenko-axon) [inline bladeui/css/blade.scss]: this page uses in registration

Comment (maksym-dyachenko-axon) [reply bladeui/css/blade.scss]: this section name is very strange, please divide this files to appropriate folders
You can leave here only getStarted page

Comment (iryna-novykova-axon) [reply bladeui/css/blade.scss]: this naming is after your constant name in baseClass (PAGES_ALLOWED_FOR_UNREGISTERED)

File: bladeui/css/blade.scss [hunk 2]
```diff
@@ -1,24 +1,30 @@
...
+@import 'settings/transfers.scss';
+@import 'settings/profile.scss';
+
+@import 'transfers/manualTransfer.scss';
+@import 'transfers/transfersList.scss';
+
+@import 'pagesAllowedForUnregistered/getStarted.scss';
+
+@import 'registration/createPassword.scss';
+@import 'registration/recoverPhrase.scss';
```
Comment (maksym-dyachenko-axon) [inline bladeui/css/blade.scss]: this one should be in recovery section

**Pull request merged**